src/Wizard/Wizard.jsx

```diff
@@ -18,12 +18,23 @@ export class Wizard extends React.Component {
     this.state = _.assign(INITIAL_STATE, {
       data: props.initialWizardData || {},
     });
+
+    if (this.props.initialStep) {
+      this.setState({currentStep: this.props.initialStep});
```

Comment:
You should't call `this.setState` in the constructor - just mutate `this.state` directly